### ISSUE TITLE
Upload actions are missing focus outline

### ISSUE THREAD

**Issue by JammingBen** [opencloud-eu/web#2002]
### Initial Checklist

- [x] I understand this is a bug report and questions should be posted in the [Community Discussions](https://github.com/orgs/opencloud-eu/discussions/)
- [x] I searched [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Bug Description

The upload actions in the "New" menu are missing their focus outline when navigating via tab.

### Reproduction Steps

- Open the "New" menu
- Navigate through the items using tab

### Expected Outcome

Every selected items gets a visible outline.

### Actual Outcome

The outline is missing for the file- and folder upload action.

<img width="314" height="212" alt="Image" src="https://github.com/user-attachments/assets/29b95d1e-16b0-4abd-b482-668aa0a12455" />

**Comment by JammingBen** [comment]
Has been fixed meanwhile.
